staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -37,6 +37,13 @@ import (
 	utiltrace "k8s.io/utils/trace"
 )
 
+const (
+	// ValidatingWebhookFailedOpenKeySuffix in an annotation indicates the
+	// validating webhook failed open when the webhook backend connection
+	// failed or returned an internal server error.
+	ValidatingWebhookFailedOpenKeySuffix string = "failed-open"
```

Comment:
is there a reason to limit this to validating webhooks and not annotate for mutating webhooks that fail open as well?

Reply:
no particular reason, it's just that GCP Binauthz only uses the validating webhook.  I can try to add the annotation to mutating webhooks.

Reply:
Added fail open log to mutation webhook as well.